src/features/projectsV2/ProjectDetails/components/microComponents/TreeMapperBrand.tsx

```diff
@@ -0,0 +1,22 @@
+import TreeMapperIcon from '../../../../../../public/assets/images/icons/projectV2/TreeMapperIcon';
+import styles from '../../styles/PlantLocationInfo.module.scss';
+import { useTranslations } from 'next-intl';
+
+const TreeMapperBrand = () => {
+  const tProjectDetails = useTranslations('ProjectDetails');
+  return (
+    <div className={styles.treeMapperLabelContainer}>
+      <div className={styles.treeMapperLabelSubContainer}>
+        <p>{tProjectDetails('poweredBy')}</p>
+        <div>
+          <TreeMapperIcon />
+        </div>
+        <p className={styles.treeMapperLabel}>
+          {tProjectDetails('treeMapper')}
+        </p>
```

Comment:
Instead of splitting this into 3 different parts, you can use `next-intl`s rich text feature 

e.g.

`"testRichReact": "This is the <iconText>IconText</iconText>"`

```
{tProjectDetails.rich('testRichReact', {
        iconText: (chunks) => (
          <div className={styles.treeMapperLabel}>
            <TreeMapperIcon />
            {chunks}
          </div>
        ),
      })}
```